### DISCUSSION TITLE
PVP Gear Requirements

### DISCUSSION THREAD

**Discussion by Erson33** [chromiecraft/chromiecraft#4635] [Suggestions]
With TBC launching I think it would be prudent to decrease the current PVP gear requirements. For newer players 10k HK is quite steep for gear that is, in many cases, on par with what you can get in dungeons. PVP needs to be more accessible to increase player numbers, and having all good rewards behind such a steep wall is not helpful.

**Comment by 55Honey** [comment]
Hello Erson33,
thanks for your suggestion.
We have removed the requirement of a certain order when buying rank 12-14 PvP epics for the start of TBC. Further adjustments will be made as we proceed. I suggest having a look at the #announcements channel as we're going forward :)

**Reply by Erson33** [reply]
Thank you for the reply, I do think that was a good start however the sheer number of kills remains steep for newer players and is quite the barrier to overcome.
